FourLoopMVC/WebContent/WEB-INF/views/addEmployee.ftl

```diff
@@ -42,39 +42,41 @@
 			<hr />
 
 			<h5>Add an employee:</h5>
-			<form method="post">
+
+			<form action="insertEmployee.mvc" method="post">
+
 				<div class="row">
 					<div class="large-6 columns">
 						<label>First Name</label>
```

Comment:
You should put a for attribute in here, this will indicate the label belongs to the element.